pkg/cli/cli.go

```diff
@@ -424,7 +442,7 @@ After the scaffold is written, api will run make on the project.
   make run
 `,
 			c.commandName, c.commandName),
-
+		Version: c.version,
```

Comment:
According to the docs, this is only necessary if the command doesn't define a `version` command.

Reply:
According to [the docs](https://godoc.org/github.com/spf13/cobra#Command) this has to do with `-v` and `--version` flags, not with `version` command:
>// Version defines the version for this command. If this value is non-empty and the command does not
>// define a "version" flag, a "version" boolean flag will be added to the command and, if specified,
>// will print content of the "Version" variable. A shorthand "v" flag will also be added if the
>// command does not define one.

Reply:
Ah I read that wrong. I don't think we want to expose a `--version` _and_ a `version` command though.

Reply:
I don't see any damage done by being able to do `kubebuilder version`, `kubebuilder --version` and `kubebuilder -v`. I think it actually improves UX as it doesn't have to know if it is a command or a flag. `kubebuilder help` and `kubebuilder --help` have the same behavior, right?